infra-as-code/bicep/acr.bicep

```diff
@@ -56,20 +82,41 @@ resource acrResource 'Microsoft.ContainerRegistry/registries@2023-07-01' = {
     zoneRedundancy: 'Enabled'
   }
 
-  // TODO (P2 - Jon): Add a build agent node connected to the virtual network, the image building subnet.
+  // TODO (P2 - Jen): Add a build agent node connected to the virtual network, the image building subnet.
   // Then pushes will happen from within the network when following the instructions in the README.
-  // TODO (P2 - Jon): I believe the user is going to need AcrPush in order to put the new image into ACR.
-
-  //resource x 'agentPools@2019-06-01-preview' = {
-  //  name: 'sdf'
-  //  properties: {
-  //    os: 'Linux'
-  //    count: 1
-  //    virtualNetworkSubnetResourceId: // TODO
-  //  }
-  //}
+  // TODO (P2 - Jen): I believe the user is going to need AcrPush in order to put the new image into ACR.
 }
 
+resource acrBuildAgentNode 'Microsoft.ContainerRegistry/registries/agentPools@2019-06-01-preview' = {
+  name: acrBuildAgentNodeName

```

Comment:
Please hardcode this `name`

Reply:
Do you want me to hard code a GUID? The "name" has to be a guid

Reply:
Nah, just a regular alpha resource name is fine. I already have a name for this in this branch, so let's stick with that unless you have something different that is better.